civictechprojects/static/css/partials/_AboutProject.scss

```diff
@@ -3,8 +3,7 @@
 
 .AboutProjects-root {
     display: flex;
-    width: 96vw;
-    margin: auto;
+    margin: 0 40px;
```

Comment:
Now the right side of the screen has the extra space.

![image](https://user-images.githubusercontent.com/322346/59320649-f6c1a100-8c83-11e9-8e5b-fe292c59e838.png)